pkg/permissionclaim/permissionclaim_labeler.go

```diff
@@ -131,3 +135,29 @@ func (l *Labeler) LabelsFor(ctx context.Context, cluster logicalcluster.Name, gr
 
 	return labels, nil
 }
+
+func isSelected(claim apisv1alpha1.PermissionClaim, name, namespace string) bool {
```

Comment:
@nrb do you mind to export this function? As it turns out the solution suggested in https://github.com/kcp-dev/kcp/pull/2456/files#r1065754904 won't fly. After discussing OOB with @ncdc and @sttts we figured the step to authorize create requests must go into admission. There, I'd reuse this function. 🙏

Reply:
Yep, I can do that.

Reply:
Done